holoviews/element/tiles.py

```diff
@@ -127,27 +126,10 @@ def easting_northing_to_lon_lat(easting, northing):
     )
 }
 
-def deprecation_warning(name, url, reason):
-    def deprecated_tilesource_warning():
```

Comment:
This deprecation approach for tile sources was added not so long ago...removing it now might be a bit too soon!

Reply:
This was added two years ago is that too soon?

https://github.com/holoviz/holoviews/pull/4941

Reply:
The notebook is working. Though, I update the text to reflect it.

Reply:
Thanks!